api/controllers/PostController.js

```diff
@@ -54,6 +54,12 @@ const fetchAndPresentPosts = function (query, userId, relationsOpts) {
     data.posts.forEach((post, i) => normalizePost(post, buckets, i === data.posts.length - 1))
     return Object.assign(data, buckets)
   })
+  .then(data =>
```

Comment:
if you do this with `tap` then it's a bit briefer -- you can skip `return data` at the end